lib/vectortilefeature.js

```diff
@@ -192,6 +197,57 @@ VectorTileFeature.prototype.toGeoJSON = function(x, y, z) {
     return result;
 };
 
+VectorTileFeature.prototype.toOffsetsJSON = function() {
+    // Resize tiles to 512x512 bounding box
+    var tileSize = 512;
+    var tileExtent = this.extent;
+    var transformFunction = function (x, y) {
+        return [
+            (x/tileExtent) * tileSize,
+            (y/tileExtent) * tileSize
+        ];
+    };
+
+    var type = VectorTileFeature.types[this.type];
+    var geometryOffsets = this.loadGeometry(transformFunction);
+    var i;
+
+    switch (this.type) {
+        case 1:
+            var points = [];
+            for (i = 0; i < geometryOffsets.length; i++) {
+                points[i] = geometryOffsets[i][0];
+            }
+            geometryOffsets = points;
+            break;
+
+        case 3:
+            geometryOffsets = classifyRings(geometryOffsets);
+            break;
+    }
+
+    if (geometryOffsets.length === 1) {
+        geometryOffsets = geometryOffsets[0];
+    } else {
+        type = 'Multi' + type;
+    }
+
+    var jsonFeature = {
+        type: "Feature",
+        properties: this.properties,
+        geometry: {
+            type: type,
+            coordinates: geometryOffsets
+        }
+    };
+
+    if ('id' in this) {
```

Comment:
This is ignorance, but can't the 'id' be another value?

Reply:
I don't get what you mean :(

Reply:
I mean 'id' can't be 'ID' or 'Id' or any other value, right? --> 'id' comes from spec?

Reply:
Yes, that comes from [MVT spec](https://github.com/mapbox/vector-tile-spec/tree/master/2.1#42-features). It is not required, but that will be helpful for interactivity, given that clipping won't merge features by id.